x-pack/plugins/security_solution/public/overview/components/detection_response/host_alerts_table/host_alerts_table.tsx

```diff
@@ -117,41 +108,53 @@ const getTableColumns: GetTableColumns = ({ getAppUrl, navigateTo }) => [
     field: 'totalAlerts',
     name: i18n.ALERTS_TEXT,
     'data-test-subj': 'hostSeverityAlertsTable-totalAlerts',
-    render: (totalAlerts: number) => <FormattedCount count={totalAlerts} />,
+    render: (totalAlerts: number, { hostName }) => (
+      <EuiLink onClick={() => handleClick({ hostName })}>
```

Comment:
can you store this anonymous function and the others in this file in a `useCallback`?

Reply:
fixed :D. Thank you!